### PR TITLE
Adding to README.md

### PR DESCRIPTION
**Pull request by Fluffpachirisu**
A little step that the console tells you to do and it's obvious if you have used Node.js but still, no harm on adding it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,10 @@ Copy `config/config-example.js` into `config/config.js`, and edit as you please.
 
 Congratulations, you're done setting up Pokémon Showdown.
 
+After you will need to install the dependencies:
+
+    npm install
```
Comment (panpawn) [inline README.md]: Isn't this is included in ``pokemon-showdown``?

Comment (Fluffpachirisu) [inline README.md]: No, the console tells you when you type "node pokemon-showdown" to start it if the their not installed. But no harm on having it on the file.

Comment (Zarel) [inline README.md]: That's probably specific to the Windows code. Why can't the Windows code auto-install? @Morfent?

Comment (Zarel) [inline README.md]: Anyway, this should ideally be fixed by fixing the code to auto-install dependencies, not by amending the readme.

Comment (Morfent) [inline README.md]: It already does that,  I think

Comment (Morfent) [inline README.md]: https://github.com/Zarel/Pokemon-Showdown/blob/master/pokemon-showdown#L18

**Comment by Zarel** [comment]
Okay, I'm now very confused.

Can you paste the error message you get when the console tells you to do `npm install`? It's not supposed to ever do that.

**Comment by Morfent** [comment]
If `npm install` isn't working on Windows, I'll get a VM for it set up again and see why

**Comment by panpawn** [comment]
I'm pretty sure I've used ``npm install`` on Windows before

**Comment by Morfent** [comment]
I meant doing it from running `pokemon-showdown`

**Comment by QuiteQuiet** [comment]
Did some reading up on this, and found out https://github.com/Zarel/Pokemon-Showdown/blob/master/pokemon-showdown#L22 `npm` will only run correctly if it's used as `child_process.spawnSync('npm.cmd', ['install', '--production'], {stdio: 'inherit'});` on Windows.

**Comment by Zarel** [comment]
That seems like a relatively simple fix. Can you commit it?

**Comment by QuiteQuiet** [comment]
Does Linux versions/OS X accept `npm.cmd` or is something more like `process.platform === 'win32' ? 'npm.cmd' : 'npm'` needed? I can, but I don't know how they would deal with it.

**Comment by Zarel** [comment]
Superseded by #3465

**Pull request closed without merging**